libgraph/include/katana/PropertyGraph.h

```diff
@@ -997,6 +1010,38 @@ class KATANA_EXPORT PropertyGraph {
   Result<void> WriteView(
       const std::string& uri, const std::string& command_line);
 
+  /// Bitmask of nodes included in the transformation view. Should be used to construct arrow tables.
+  std::shared_ptr<arrow::Buffer> NodeBitmask() const noexcept {
+    if (is_transformed) {
+      auto length = original_to_transformed_nodes_.size();
+      return std::make_shared<arrow::Buffer>(
+          node_bitmask_data_.data(), arrow::BitUtil::BytesForBits(length));
+    }
+    return nullptr;
+  }
+
+  /// Bitmask of edges included in the transformation view. Should be used to construct arrow tables.
+  std::shared_ptr<arrow::Buffer> EdgeBitmask() const noexcept {
+    if (is_transformed) {
+      auto length = original_to_transformed_edges_.size();
+      return std::make_shared<arrow::Buffer>(
+          edge_bitmask_data_.data(), arrow::BitUtil::BytesForBits(length));
+    }
+    return nullptr;
+  }
+
+  /// Return the number of nodes of the original property graph.
+  uint64_t NumOriginalNodes() const {
```

Comment:
why not use this logic in `NumNodes`? Asking because I'd like to understand why we'd like to keep the original `NumNodes` as is.

Reply:
`NumNodes` is the number of nodes in the topology, which is smaller for projections. We'd need it when we want to iterate over all nodes in the topology. This `NumOriginalNodes` is really only used when adding a new node property to the table, to make sure I allocate the correct number of rows.